pantheon-bundle/src/test/java/com/redhat/pantheon/servlet/AssemblyRenderingServletTest.java

```diff
@@ -0,0 +1,259 @@
+package com.redhat.pantheon.servlet;
+
+import com.redhat.pantheon.asciidoctor.AsciidoctorService;
+import com.redhat.pantheon.model.module.Module;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.testing.mock.sling.ResourceResolverType;
+import org.apache.sling.testing.mock.sling.junit5.SlingContext;
+import org.apache.sling.testing.mock.sling.junit5.SlingContextExtension;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import javax.servlet.http.HttpServletResponse;
+import java.util.Locale;
+
+import static com.redhat.pantheon.util.TestUtils.registerMockAdapter;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.*;
+
+@ExtendWith({SlingContextExtension.class, MockitoExtension.class})
+public class AssemblyRenderingServletTest {
```

Comment:
A better name for this test would be `AssemblyRenderingTest`

Reply:
As per Ben's  and Carlos's suggestion Changed name to noun ( AssemblyRenderer)